### ISSUE TITLE
[flow] Wrong type for Popup#position and Map#doubleClickZoom

### ISSUE THREAD

**Issue by ghostd** [PaulLeCam/react-leaflet#370]
Hi,

Accoding to [the code](https://github.com/PaulLeCam/react-leaflet/blob/master/src/Popup.js#L76), i think the [Popup#position type](https://github.com/PaulLeCam/react-leaflet/blob/master/src/Popup.js#L21) could be `?position: LatLng`

Currently i simply call `<Popup>some content</Popup>` and all works, but `flow` (0.53) is crying:
```
241:             <Popup>
                 ^^^^^^^ React element `Popup`
 24: export default class Popup extends MapComponent<LeafletElement, Props> {
                                                                     ^^^^^ property `position`. Property not found in. See: node_modules/react-leaflet/lib/Popup.js.flow:24
241:             <Popup>
                 ^^^^^^^ React element `Popup`
```

I get the same issue for [Map#doubleClickZoom](https://github.com/PaulLeCam/react-leaflet/blob/d994aac47d35cc36a6dbdd6e964385da99053db2/src/Map.js#L58), since `leaflet` handle a [default value](http://leafletjs.com/reference-1.2.0.html#map-doubleclickzoom)

**Comment by PaulLeCam** [comment]
Thanks for reporting this, should be fixed in v1.6.5.

**Comment by ghostd** [comment]
Thanks, the errors are fixed with the new version.
